indigo_api/models/tasks.py

```diff
@@ -359,6 +384,10 @@ def update_blocked_tasks(self, task, user):
             for blocked_task in blocked_tasks:
                 action.send(user, verb='resolved a task previously blocking', action_object=blocked_task,
                             target=task, place_code=blocked_task.place.place_code)
+                action.send(user, verb='unblocked', action_object=task,
```

Comment:
why would we do this and then only conditionally actually call `blocked_task.unblock`?